reconcile/change_owners/change_owners.py

```diff
@@ -390,5 +439,11 @@ def run(
                 # skips MRs with this label
                 gl.remove_label(merge_request, SELF_SERVICEABLE)
 
+            if not change_admitted:
+                raise NotAdmittedError("Change not admitted")
+
+    except NotAdmittedError as e:
+        logging.error(e)
+        sys.exit(1)
```

Comment:
We want to fail here because we assume this is the first integration that is going to run and we don't want to move on with the rest of the integrations, correct?

Reply:
exactly

Reply:
Nice, thanks. A comment explaining this would be very helpful, especially because it express an expectation on how integrations are being run via hack scripts.

Reply:
added